internal/controllers/reward.go

```diff
@@ -0,0 +1,12 @@
+package controllers
+
+import (
+	"coursebench-backend/internal/controllers/reward"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+func RewardRoutes(r fiber.Router) {
```

Comment:
Add a call to this function in `internal/fiber/route.go`, otherwise it won't be routable.

Reply:
fixed

Reply:
Please change the commit message to include `internal/fiber.Routes`: `fix(internal/fiber.Routes): add call to controllers.RewardRoutes`